cvat-ui/src/reducers/review-reducer.ts

```diff
@@ -197,7 +197,7 @@ export default function (state: ReviewState = defaultState, action: any): Review
         }
         case AnnotationActionTypes.CLOSE_JOB:
         case AuthActionTypes.LOGOUT_SUCCESS: {
-            return { ...defaultState };
+            return defaultState;
```

Comment:
Should we apply such approach for all the default state actions? We have plenty of `return { ...defaultState }` in other places